webui.py

```diff
@@ -1036,22 +1036,21 @@ def simple_body(self, path):
             raise NotFound, path + " does not have any releases"
 
         html = []
-        html.append("""<html><head><title>Links for %s</title><meta name="api-version" value="2" /></head>"""
+        html.append("""<!DOCTYPE html><html><head><title>Links for %s</title></head>"""
```

Comment:
The `!DOCTYPE` addition is sound, however I'd like to stage it in a separate PR if possible.